html/pages/device.inc.php

```diff
@@ -229,6 +229,14 @@
                 </a>
               </li>');
 
+        if (@dbFetchCell("SELECT COUNT(stp_id) FROM stp WHERE device_id = '".$device['device_id']."'") > '0') {
```

Comment:
I don't think that we would have more then one STP/RSTP instance per device, maybe in VRF.